packages/server/src/controllers/apikey/index.ts

```diff
@@ -61,9 +61,6 @@ const verifyApiKey = async (req: Request, res: Response, next: NextFunction) =>
             new ApiError(StatusCodes.PRECONDITION_FAILED, `Error: apikeyController.verifyApiKey - apiKey not provided!`)
```

Comment:
should we `throw new ApiError` or just `throw new Error`? Since the other controllers are just throwing Error

Reply:
We now have the same error throwing mechanism in the entire server package